doc/data/messages/n/non-ascii-file-name/pylintrc

```diff
@@ -0,0 +1,2 @@
+[main]
+py-version = 3.4
```

Comment:
Do we still need that if there's no example ? Also I think the intent of the check is to be activated even in python 3.5+

Reply:
The thing that was agreed on was that either it would be removed or add a py-version checker. Since the alert is not going to be removed, I am adding a py-version checker.

Reply:
Yeah sorry about the confusion, I did not check the reasoning behind the check in detail at first. The comment in the code made me reevaluate what we should do (check even in python 3.5+ for interoperability reasons).

Reply:
Ok, so should I change the py-version to 3.5 or something above 3.5?

Reply:
No py-version / pylintrc at all if possible.

Reply:
Ok, I've removed the pylintrc